x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class IndexPattern {
```

Comment:
You can export these w/o a class
```
export const eventsIndex = 'events-endpoint-*';
```
and you can import the namespace like this:
```
import { eventsIndex } from 'blah'
// or like this
import * as indexPatterns from 'blah'
``then use them like `indexPatterns.eventsIndex`
```

Reply:
thanks I'll update them